frontend/ui/widget/keyvaluepage.lua

```diff
@@ -543,6 +544,90 @@ function KeyValuePage:_populateItems()
     self.return_button:resetLayout()
     self.main_content:clear()
     local idx_offset = (self.show_page - 1) * self.items_per_page
+
+    -- for flexible middle ratio calculation
+    local frame_padding = Size.padding.default
+    local frame_internal_width = self.item_width - frame_padding * 2
+    local middle_padding = Size.padding.default -- min enforced padding between key and value
+    local available_width = frame_internal_width - middle_padding
```

Comment:
May be add a comment here (KeyValuePage) and there (KeyValueItem) that these paddings values and computations should be kept in sync.
(Or put frame_padding and middle_padding as default properties in `KeyValueItem = InputContainer:extend{}`, and use them here `local frame_padding = KeyValueItem.frame_padding`)

Reply:
Adopted the latter solution.